test/async-hooks/test-statwatcher.mjs

```diff
@@ -0,0 +1,105 @@
+import { isMainThread, skip, platformTimeout, mustCallAtLeast } from '../common/index.mjs';
+import { refresh, path as _path } from '../common/tmpdir.js';
+import { strictEqual } from 'assert';
+import initHooks from './init-hooks.mjs';
+import { checkInvocations } from './hook-checks.mjs';
+import { watchFile, writeFileSync, unwatchFile } from 'fs';
+import { join } from 'path';
+import process from 'process';
+
+if (!isMainThread)
+  skip('Worker bootstrapping works differently -> different async IDs');
+
+refresh();
+
+const file1 = join(_path, 'file1');
+const file2 = join(_path, 'file2');
```

Comment:
same here, `_path` means nothing to me, while `tmpdir.path` is immediately obvious what it is.
```suggestion
const file1 = join(tmpdir.path, 'file1');
const file2 = join(tmpdir.path, 'file2');
```